modules/apiserversource-kn.adoc

```diff
@@ -113,10 +136,34 @@ Data,
 [id="delete-apiserversource-kn_{context}"]
 == Deleting the ApiServerSource
```

Comment:
Is this a separate procedure from Creating the ApiServerSource? If so, it could be it's own procedure module.